### PR TITLE
Modified adobe-brackets cheat sheet

### PR DESCRIPTION
**Pull request by navneetzz**
added proper description to this cheat sheet and modified the way cheat sheet was displayed.
 before - 
brackets o
brackets o2

after - 
brackets 1

https://duck.co/ia/view/adobe_brackets_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/adobe-brackets.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "id": "adobe_brackets_cheat_sheet",
     "name": "Adobe Brackets",
-    "description": "Text Editor",
+    "description": "A modern, open source text editor from Adobe",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/adobe-brackets.json]: @navneetzz  lets make this "Keyboard shortcuts for Brackets, an open source text editor from Adobe"

**Comment by jophab** [comment]
@navneetzz  At line 61 remove the brackets around F2

**Comment by moollaza** [comment]
@navneetzz @jophab  Thanks! This LGTM :+1:

**Pull request merged**